cfgov/jinja2/v1/events/event.html

```diff
@@ -1,8 +1,8 @@
 {% extends 'layout-2-1-bleedbar.html' %}
 {% import '_vars-events.html' as vars with context %}
 {% import 'templates/streamfield-sidefoot.html' as streamfield_sidefoot with context %}
-{% if page.start_dt and page.end_dt %}
-    {% set event_state = when(page.start_dt | date('%c'), page.end_dt | date('%c')) %}
+{% if page.live_stream_date and page.end_dt %}
+    {% set event_state = when(page.live_stream_date, page.end_dt) %}
```

Comment:
We may need to have a separate variable called `live_stream_event_state` and revert this particular change.
Also a frequent content user error is to check `live_stream_availability` and not enter a date then they end up with 500 errors when the template looks for `page.live_stream_date`. 
so either we'll need to wrap the  `page.live_stream_date` in a conditional that checks for  `live_stream_availability` first or add a form validation check